compiler/main.nim

```diff
@@ -204,11 +211,12 @@ proc mainCommand*(graph: ModuleGraph) =
       commandCompileToC(graph)
     else:
       rawMessage(conf, errGenerated, "'run' command not available; rebuild with -d:tinyc")
-  of "js", "compiletojs":
```

Comment:
why was `compiletojs` removed?

Reply:
Oops, introducing it again

Reply:
feel free to click "resolved" when you've addressed a comment (unless for cases where the way you've resolved it could potentially be controversial; obviously not the case here)